packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -249,7 +251,7 @@ class DescopeWc extends BaseDescopeWc {
     }
 
     // if there is no screen id (probably due to page refresh) we should get it from the server
-    if (!screenId && !startScreenId) {
+    if (!screenId && !startScreenId && !oidcIdpStateId) {
```

Comment:
Why do we want to change this condition?

Reply:
oh, its a leftover from previous change, removed 👍